test/helpers/sign.js

```diff
@@ -9,6 +9,15 @@ function toEthSignedMessageHash (messageHex) {
   return web3.utils.sha3(Buffer.concat([prefix, messageBuffer]));
 }
 
+function fixSignature (signature) {
+  //in geth its always 27/28, in ganache its 0/1. Change to 27/28 to prevent
+  //signature malleability
+  //https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L447
```

Comment:
```suggestion
  // see https://github.com/ethereum/go-ethereum/blob/v1.8.23/internal/ethapi/api.go#L465
```
I changed the reference from `master` to the latest release, to prevent this link from becoming invalid in the future.

Reply:
makes sense